AGPs/AGP-119.md

```diff
@@ -2,7 +2,7 @@
 AGP: 119
 Title: Authorship Rewards Program
 Author: burrrata (@burrrata)
-Status: Stage V
+Status: Rejected
```

Comment:
This is consistent with how we have updated proposals which were rejected during the Association Review phase previously so there is no issue here... However, it does seem like it might be helpful to make that distinction.